client/src/App.js

```diff
@@ -3,18 +3,26 @@ import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+import Home from "./pages/Home";
+import Referral from "./pages/Referral";
 
 import "./App.css";
 
-function App() {
-  return (
-    <MuiThemeProvider theme={theme}>
-      <BrowserRouter>
-        <Route path="/" component={LandingPage} />
-      </BrowserRouter>
-    </MuiThemeProvider>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <MuiThemeProvider theme={theme}>
+        <BrowserRouter>
+          <Route path="/home" component={Home} />
```

Comment:
You still want the path "/" to take the user home (and later, to the Signup page, if they're logged out)